src/mixins/java/org/spongepowered/common/mixin/api/minecraft/server/MinecraftServerMixin_API.java

```diff
@@ -350,6 +350,11 @@ public GameProfileManager gameProfileManager() {
         return this.api$profileManager;
     }
 
+    @Override
+    public SpongeGameProfileManager gameProfileManagerIfPresent() {
```

Comment:
Just `gameProfileManager`

Reply:
There's already a `gameProfileManager()` method (just above), that creates the game profile manager when its called. for shutdown purposes, we don't want to create a new manager instance though, which is why this method exists.